### ISSUE TITLE
Target stats in Robust BatchNorm Layer

### ISSUE THREAD

**Issue by MarcLafon** [BIT-DA/RoTTA#8]
Hello, I was wondering what is the purpose of `target_mean` and `target_var` line 18 and 19 in the bn_layers.py file: [https://github.com/BIT-DA/RoTTA/blob/67e34c900cdd355fc07e55edd4c577ea7b8ebcc9/core/utils/bn_layers.py#L18](url)

It seems that these variables are not used. Same for `current_mu` and `current_sigma`. 

Additionally could you give more details on how the behavior of the Robust Batch Norm differs from the one of the normal Batch Norm (which also update its stats with moving average).

Great work by the way!

**Comment by yuanlonghui** [comment]
Thanks for your interest on our work. The variables are merely meaningless ones defined in the base class (compared to RBN, these variables might have been used in some of my experimental variants), so please ignore them.

The main difference between Robust Batch Norm and normal Batch Norm is which sample is used for updating the normalization statistics. We have clearly described them in the main paper. For the RBN, we only use the samples in the memory bank for statistics updating.

**Comment by MarcLafon** [comment]
Thank your for your explanation it is clear now! I guess I got confused with the unused variables.
